metasploit-resource-portal/source/_topnav.erb

```diff
@@ -1,9 +1,8 @@
 <% nav as: :pills, position: :static do %>
-  <%= link_to 'Docs',      'https://help.metasploit.com' %>
-  <%= link_to 'API',       'https://rapid7.github.io/metasploit-framework/api/' %>
-  <%= link_to 'Community', 'https://community.rapid7.com/community/metasploit' %>
-  <%= link_to 'Support',   'https://www.rapid7.com/support/' %>
-  <%= link_to 'Stats',     'pull_requests.html' %>
+  <li><a href="https://help.metasploit.com">Docs</a></li>
```

Comment:
Existing issue, looks like this cert has recently expired

<img width="711" alt="image" src="https://user-images.githubusercontent.com/60357436/167962904-85d7c068-5db2-4572-8739-c4035b4a98b8.png">

I think it's meant to be `https://docs.rapid7.com/metasploit/`